polaris-core/src/main/java/org/apache/polaris/core/storage/azure/AzureLocation.java

```diff
@@ -28,9 +28,13 @@ public class AzureLocation extends StorageLocation {
   private static final Pattern URI_PATTERN = Pattern.compile("^(abfss?|wasbs?)://([^/?#]+)(.*)?$");
 
   public static final String ADLS_ENDPOINT = "dfs.core.windows.net";
-
   public static final String BLOB_ENDPOINT = "blob.core.windows.net";
 
+  public static final String ABFS_SCHEME = "abfs://";
+  public static final String ABFSS_SCHEME = "abfss://";
+  public static final String WASB_SCHEME = "wasb://";
+  public static final String WASBS_SCHEME = "wasbs://";
+
```

Comment:
[Typically](https://hadoop.apache.org/docs/r2.7.7/api/org/apache/hadoop/fs/s3/S3FileSystem.html#getScheme()) the `scheme` is not considered to include the `://`. I also think there might be a more extensible way to capture this than 4 schemes. 

Furthermore, isn't this duplicative of the constants in `WasbTranslatingFileIO` as well as `StorageType`?